ci/request_pr_exp.py

```diff
@@ -294,6 +294,36 @@ def _get_gke_credential(args: argparse.Namespace):
     logging.error('Failed to authenticate gcloud: %s', e)
 
 
+def _build_image(args: argparse.Namespace):
```

Comment:
I think we can just do this in https://github.com/google/oss-fuzz-gen/blob/main/ci/cloudbuild.yaml pretty easily. 

see e.g. https://github.com/google/oss-fuzz/blob/7f138b90225a76beb5ee47ebfc5cf12d624ca47e/infra/experimental/chronos/cloudbuild.yaml#L30

Reply:
Yep I did that in the first two commits.
Later commits moved these steps to the python script because I am unsure if we want to build an image for every commit/PR:
1. Some commits pushed to PR are not yet ready for experiments. Having those steps in `cloudbuild.yaml` automatically [builds images](https://github.com/google/oss-fuzz-gen/runs/32642001417) for them and wastes computation power.
2. Some PRs from attackers may contain security threats. It might not be a good idea to automatically build and push the image and show the output in CI page.

Moving the steps to `ci/request_pr_exp.py` ensures an image won't be built unless a maintainer+ role manually executes the `gcbrun` command.

Reply:
`cloudbuild.yaml` only runs with /gcbrun right? So this should never be a concern?

Reply:
i.e. we just move the docker building into `cloudbuild.yaml`, and then the image only gets build prior to the epxeriment running if someone with privileges runs `/gcbrun`.